‏‏assignment3/main.py

```diff
@@ -0,0 +1,66 @@
+import pickle as pck
+import os
+from datetime import datetime as dt
+import constants as cons
+import base64
+
+
+def load_pickle_file():
+    source = cons.PICKLE_FULL_PATH
+    result_dict = {}
+
+    with open(source, 'rb') as file:
+        loaded_pickle_file = pck.load(file)
+    for key in loaded_pickle_file.keys():
+        result_dict[base64.b32decode(key).decode('utf-8')] = loaded_pickle_file[key]
+
+    return result_dict
+
+
+def the_filename_without_suffix(filename, suffix_size):
```

Comment:
don't start functions or variables with 'the'

Reply:
Ok